src/cli/upgrade.go

```diff
@@ -63,7 +64,8 @@ var upgradeCmd = &cobra.Command{
 
 		cache.Init(sh, cache.Persist)
 
-		if _, OK := cache.Get[string](cache.Device, upgrade.CACHEKEY); OK && !force {
+		// Only respect the cache interval when using --auto flag
+		if _, OK := cache.Get[string](cache.Device, upgrade.CACHEKEY); OK && auto && !force {
```

Comment:
We can remove force here now, it's not relevant anymore as we handle that downstream.